content/themes/semantic-ui/twitter/index.slim

```diff
@@ -124,4 +124,3 @@
       == @items['/partials/examples/transition.slim'].compiled_content
 
 link href="/static/kitchen-sink/stylesheets/kitchen-sink.css" rel="stylesheet" type="text/css"
-script src="/static/kitchen-sink/javascripts/kitchen-sink.js"
```

Comment:
Still prefer put `kitchen-sink.js` to just themes pages. Later we may add some more pages, such as blog posts, templates, landing page, etc. And not all page need a `kitchen-sink.js` link.